NorthstarDLL/util/printmaps.cpp

```diff
@@ -182,10 +184,32 @@ void ConCommand_maps(const CCommand& args)
 			spdlog::info("({}) {}", PrintMapSource.at(map.source), map.name);
 }
 
+void ConCommand_map(const CCommand& args)
+{
+	if (args.ArgC() > 1)
+	{
+		RefreshMapList();
```

Comment:
Why is this required?

Reply:
Refreshes the list of maps we can load. Imo this should be done in ModManager though that would be a separate PR